source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResultClient/MeteringPointType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResultClient;
+
+/// <summary>
+/// Defines metering point type
+/// </summary>
+public enum MeteringPointType
+{
+    Consumption = 0, // E17
+    Production = 1, // E18
+    Exchange = 2, // E20
```

Comment:
I don't think it's particularly interesting to mention the eBIX names here.
```suggestion
    Consumption = 0,
    Production = 1,
    Exchange = 2,
```